### DISCUSSION TITLE
Question : Jquery

### DISCUSSION THREAD

**Discussion by aguiarcc** [pyodide/pyodide#3393] [Q&A]
Is there a way to use the Jquery Framework with Pyodide ?

**Comment by hoodmane** [comment]
It should be possible. If you try and run into trouble feel free to ask more specific questions.

**Reply by aguiarcc** [reply]
Right, i will try.

**Comment by simonegiacomelli** [comment]
FYI I was able to use the following through Pyodide:
- d3js
- Web Bluetooth api
